namespace java thrift.test

struct myKeyStruct {
  1: map<list<i32>, string> a,
}
